apps/cfp/views.py

```diff
@@ -97,7 +98,7 @@ class YouthWorkshopProposalForm(ProposalForm):
     attendees = StringField("Attendees", [DataRequired()])
     cost = StringField("Cost per attendee")
     participant_equipment = StringField("Attendee equipment")
-    age_range = StringField("Age range")
+    age_range = SelectField("Age range", choices=AGE_RANGE_OPTIONS)
```

Comment:
Maybe remove the 16+ and adult options for this case?

Reply:
Done.